gbn/gbn_conn.go

```diff
@@ -46,9 +41,7 @@ type GoBackNConn struct {
 	recvDataChan chan *PacketData
 	sendDataChan chan *PacketData
 
-	sendTimeout time.Duration
-	recvTimeout time.Duration
-	timeoutsMu  sync.RWMutex
+	timeoutsMu sync.RWMutex
```

Comment:
Do we still need this? Shouldn't all timeouts now be guarded by the timeout manager?

Reply:
Good catch!